includes/class-wc-gateway-stripe.php

```diff
@@ -647,4 +650,42 @@ public function process_payment( $order_id, $retry = true, $force_save_source =
 			);
 		}
 	}
+
+	/**
+	 * Displays the Stripe fee
+	 * @param $order_id
+	 */
+	public function display_order_fee( $order_id ) {
+		$fee = get_post_meta( $order_id, self::META_NAME_FEE, true );
+		if ( !$fee ) { return; }
+		?>
+		<tr>
+			<td class="label stripe-fee">
+				<?php echo wc_help_tip( __( 'This represents the commission fee on the Stripe transaction.', 'woocommerce-gateway-stripe' ) ); ?>
+				<?php _e( 'Stripe Fee:', 'woocommerce-gateway-stripe' ); ?></td>
+			<td width="1%"></td>
+			<td class="total">
+				-&nbsp;<?php echo wc_price( $fee, array( 'currency' => get_post_meta( $order_id, '_order_currency', true ) ) ); ?>
```

Comment:
Needs CRUD way of getting order meta with BW compat logic.

Also several lines above need escaping. And instead of `-` perhaps use an html entity?